apps/backoffice-v2/src/common/components/atoms/AssignDropdown/AssignDropdown.tsx

```diff
@@ -22,43 +23,59 @@ export const AssignDropdown: React.FC<IAssignDropdownProps> = ({
   assignees,
   avatarUrl,
   onAssigneeSelect,
-}) => (
-  <DropdownMenu>
-    <DropdownMenuTrigger asChild>
-      <div className="form-control flex w-[200px] cursor-pointer rounded-lg border border-neutral/10 px-4 py-[6px] text-sm theme-dark:border-neutral/60">
-        <div className="flex items-center justify-between">
-          <div className="flex items-center">
-            {assignedUser?.fullName ? (
-              <UserAvatar avatarUrl={avatarUrl} fullName={assignedUser?.fullName} />
-            ) : (
-              <UnassignedAvatarSvg className="d-[22px]" />
-            )}
-            <span className="ml-2 bg-transparent text-sm">
-              {assignedUser?.fullName ?? 'Unassigned'}
-            </span>
+  authenticatedUserId,
+}) => {
+  const sortedAssignees = useMemo(
+    () =>
+      // Sort assignees so that the authenticated user is always first
+      assignees
+        ?.slice()
+        ?.sort((a, b) =>
+          a?.id === authenticatedUserId ? -1 : b?.id === authenticatedUserId ? 1 : 0,
```

Comment:
@Omri-Levy Have you considered different behaviour?
For example
- Find current user
- sort rest of users in A-Z order
- join this alltogether,so first user always will me authenticated one and rest will be sorted in alpabetical order.

Reply:
I did! Considering the number of changes and operations doing so takes I preferred readability and simplicity.